### PR TITLE
Update SIMDe to 0.7.6

### PR DESCRIPTION
**Pull request by mr-c**
SIMDe v0.7.6

The next release of SIMDe add support for NEON's `vabal`; and is [likely to include `vpminnm` as well](https://github.com/simd-everywhere/simde/pull/1080/files#diff-8f9c70324b467cc31aeb93a0e2b273d9b09e2c1aeed5abadf7955c2c55d6aed3R41).

### CODE REVIEW

File: tools/simde_update.py [hunk 1]
```diff
@@ -67,10 +67,11 @@ def main():
       f.write(re.sub(SIMDE_FILE_HEADER_RE, r'\1\2\3', neon_h_buf, count=0, flags=re.MULTILINE))
       f.write("#undef SIMDE_ARM_NEON_A32V7_ENABLE_NATIVE_ALIASES\n")
       f.write("#undef SIMDE_ARM_NEON_A64V8_ENABLE_NATIVE_ALIASES\n")
-      print("'system/include/compat/arm_neon.h' updated")
     except Exception:
       print("error writing 'system/include/compat/arm_neon.h'")
       return 1
+  os.system("sed -i '/HEDLEY_EMSCRIPTEN_VERSION HEDLEY_VERSION_ENCODE(__EMSCRIPTEN_major__, __EMSCRIPTEN_minor__, __EMSCRIPTEN_tiny__)/i #include <emscripten/version.h>' system/include/compat/arm_neon.h")
```
Comment (sbc100) [inline tools/simde_update.py:73]: Can you do this part in python instead of forking out to sed?

**Comment by sbc100** [comment]
@seanptmaher @tlively

**Review by sbc100 — COMMENTED** [review]
Are there no test updates as a result of this?

File: tools/simde_update.py (hunk 1 above)
Comment (mr-c) [reply tools/simde_update.py]: Done!

**Comment by mr-c** [comment]
> Are there no test updates as a result of this?

Is that a question for me? I don't know how the emscripten tests work, sorry.

I see at https://app.circleci.com/pipelines/github/emscripten-core/emscripten/27635/workflows/8b53bdfe-28cb-40f2-90d3-7c5718d49bc3/jobs/631550?invite=true#step-107-2448 that `test_pthread_abort` failed. I can't see how that is related to updating the `arm_neon.h` file

**Comment by tlively** [comment]
We have fairly thorough SSE intrinsic tests, but for NEON it looks like we only have https://github.com/emscripten-core/emscripten/blob/main/test/neon/test_neon_wasm_simd.cpp, which is more of a smoke test.

I'm not sure if we have any workflow for determining support level beyond looking at the implementation of each intrinsic.

**Comment by mr-c** [comment]
Ping. There have been several SIMDe releases since this PR opened, we are now on 0.8.2; shall I update this PR to the latest version?

**Comment by sbc100** [comment]
Up to you.  I'd be happy to see this land, and then a followup, or you can do it all in one go.

@tlively does this change look good to go?

**Review by tlively — APPROVED** [review]
Yep, (rubberstamp) LGTM.

**Pull request merged**